manifests/quick-start/sso/overlays/workflow-controller-configmap.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 data:
   sso: |
     issuer: http://dex:5556/dex
+    issuerAlias: http://mydex:5556/dex
```

Comment:
Should we enable this? maybe we can keep this optional and commented out?

Reply:
good point @liafizan